### ISSUE TITLE
Going from all caps to a proper title case - acronyms

### ISSUE THREAD

**Issue by twigbranch** [granth/titleize#11]
Would be nice if there was an option to specify acronyms when going from all caps to titlecase.

**Comment by brandondrew** [comment]
Yes, this is a problem.  Here are some examples I'm running into.

`Electrical safety: AC and DC power circuits; antenna installation; RF hazards` correctly becomes `Electrical Safety: AC and DC Power Circuits; Antenna Installation; RF Hazards` (where `RF` is correct, *if* `humanize` and `underscore` options are off), but `ELECTRICAL AND RF SAFETY` wrongly becomes `Electrical and Rf Safety` (where `RF` gets written as `Rf`).

**Comment by brandondrew** [comment]
A solution to this problem should also handle things like `PhD`—not just cases where the goal is *all* caps like `NASA`.

**Comment by brandondrew** [comment]
I think it would make sense for very common cases to be included, but for it to be easy to pass in an array of strings in the target case.  For example, `AC`, `DC`, `PhD`, and `RF` might be built-in, but [`AFD`](http://www.acronymslist.com/cat/nasa-acronyms.html) might be considered too domain-specific.
